fireblocks_sdk/sdk.py

```diff
@@ -1618,8 +1618,11 @@ def _post_request(self, path, body=None, idempotency_key=None):
         response = self.http_session.post(self.base_url + path, headers=headers, json=body, timeout=self.timeout)
         return handle_response(response)
 
-    def _put_request(self, path, body=None):
+    def _put_request(self, path, body=None, query_params=None):
         body = body or {}
+        query_params = query_params or {}
```

Comment:
This is unnecessary as None is also falsy, so it won't get into the if statement

Reply:
Yup, redundant,  will fix